docs/ci-cd/index.md

```diff
@@ -35,5 +35,13 @@ Work In Progress
 ## Tools
 
 -   [Daily Fail Summary](tools/daily_fail_summary.md)
+-   Spellcheck
+    -   Uses
+        [rojopolis/spellcheck-github-actions](https://github.com/marketplace/actions/github-spellcheck-action#configuration),
+        a PySpelling-based spellchecker
+    -   This tool utilizes the definitions in .spellcheck.yml and
+        .github/.wordlist.txt to check all documentation files. .spellcheck.yml
+        defines the settings while .worklist.txt is a dictionary of words to
```

Comment:
use 

```
`.wordlist.txt`
```

to avoid spellchecking. also typo: worDlist instead of worKlist

Reply:
Added and fixed.